src/constants/typography.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components"
+import "./typefaces.css"
+
+const fonts = {
+  workSans: `'Work Sans', sans-serif`,
+  futura: `'Futura', sans-serif`,
+  futuraBold: `'Futura Bold', sans-serif`,
+  futuraHeavy: `'Futura Heavy', sans-serif`,
+}
+
+const fontSizes = {
+  h1: `calc(40px + (90 - 40) * ((100vw - 320px) / (1440 - 320)))`,
```

Comment:
When you squish this header it does this, can you give it some leading please?

![image](https://user-images.githubusercontent.com/29356116/112563024-f50c9300-8dae-11eb-94ee-db99904dfc0d.png)

Reply:
Upon further inspection this is a Safari bug, check the following recording:

https://user-images.githubusercontent.com/29356116/112563575-05713d80-8db0-11eb-9a28-750391873cd3.mov

My suspicion is that there is something weird going on with gatsby + styled components + the way styled components does browser fallbacks for CSS